managed/resource_db_query_logging.go

```diff
@@ -0,0 +1,414 @@
+
+	// fetch log config from backend
+	specList, _, err := apiClient.ClusterApi.ListPgLogExporterConfigs(ctx, accountId, projectId, clusterId).Execute()
+	if err != nil {
+		return dbAuditExporterConfig, false, GetApiErrorDetails(err)
+	}
+
+	if len(specList.GetData()) == 0 {
+		return dbAuditExporterConfig, false, fmt.Sprintf("No DB query logging config found for cluster %s", clusterName)
+	}
+
+	spec := specList.Data[0]
+	exportConfig := spec.Spec.ExportConfig
+
+	// Initialize the LogConfig object from PgLogExportConfig
+	logConfig := LogConfig{
+		LogMinDurationStatement: types.Int64{Value: int64(exportConfig.LogMinDurationStatement)},
+		DebugPrintPlan:          types.Bool{Value: exportConfig.DebugPrintPlan},
+		LogConnections:          types.Bool{Value: exportConfig.LogConnections},
+		LogDisconnections:       types.Bool{Value: exportConfig.LogDisconnections},
+		LogDuration:             types.Bool{Value: exportConfig.LogDuration},
+		LogErrorVerbosity:       types.String{Value: string(exportConfig.LogErrorVerbosity)},
+		LogStatement:            types.String{Value: string(exportConfig.LogStatement)},
+		LogMinErrorStatement:    types.String{Value: string(exportConfig.LogMinErrorStatement)},
+		LogLinePrefix:           types.String{Value: exportConfig.LogLinePrefix},
+	}
+
+	dbAuditExporterConfig = DbQueryLoggingConfig{
+		AccountID:       types.String{Value: accountId},
+		ProjectID:       types.String{Value: projectId},
+		ClusterID:       types.String{Value: clusterId},
+		ClusterName:     types.String{Value: clusterName},
+		IntegrationName: types.String{Value: integrationName},
+		State:           types.String{Value: string(spec.Info.State)},
+		ConfigID:        types.String{Value: spec.Info.Id},
+		LogConfig:       &logConfig,
+	}
+
+	return dbAuditExporterConfig, true, ""
+}
+
+func getConfigFromState(ctx context.Context, state tfsdk.State, config *DbQueryLoggingConfig) {
+	state.GetAttribute(ctx, path.Root("account_id"), &config.AccountID)
+	state.GetAttribute(ctx, path.Root("project_id"), &config.ProjectID)
+	state.GetAttribute(ctx, path.Root("cluster_name"), &config.ClusterName)
+	state.GetAttribute(ctx, path.Root("cluster_id"), &config.ClusterID)
+	state.GetAttribute(ctx, path.Root("integration_name"), &config.IntegrationName)
+	state.GetAttribute(ctx, path.Root("config_id"), &config.ConfigID)
+}
+
+func (r resourceDbQueryLogging) Read(ctx context.Context, req tfsdk.ReadResourceRequest, resp *tfsdk.ReadResourceResponse) {
+	var state DbQueryLoggingConfig
+
+	getConfigFromState(ctx, req.State, &state)
+	apiClient := r.p.client
+	accountId := state.AccountID.Value
+	projectId := state.ProjectID.Value
+	clusterId := state.ClusterID.Value
+	clusterName := state.ClusterName.Value
+	integrationName := state.IntegrationName.Value
+
+	dbe, readOK, message := resourceRead(ctx, accountId, projectId, clusterId, clusterName, integrationName, apiClient)
```

Comment:
Please improve variable naming to something more logical. Perhaps "dbqlConfig" (Just a suggestion)